module/api/src/Application/Serializer/Normalizer/ExceptionNormalizer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Api\Application\Serializer\Normalizer;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Ergonode\Api\Application\Mapper\ExceptionMapperInterface;
+use Ergonode\Api\Infrastructure\Normalizer\ExceptionNormalizerInterface;
+use Ergonode\Api\Application\Exception\FormValidationHttpException;
+use Ergonode\Api\Application\Exception\ViolationsHttpException;
+
+class ExceptionNormalizer implements NormalizerInterface
```

Comment:
I think there's already native SF functionality with this approach but might be wrong